build.sbt

```diff
@@ -17,7 +17,7 @@ lazy val core = module("core")
   .crossDepSettings(
     commonDeps ++ Seq(
       %("cats-free"),
-      %("iota-core"),
+      %("iota-core", "0.3.3"),
```

Comment:
Are this behind in org-policies? shouldn't this changes happen there?

Reply:
I've checked it, we can bump the sbt-freestyle plugin to `0.3.11` (we're currently using the `0.3.7` version) but the `0.3.11` uses the `0.3.15` version of `sbt-org-policies` where the selected `iota` version is `0.3.2`.

Two choices:
* Bump `sbt-freestyle` to `0.3.11`
* Release a new version of `sbt-freestyle` and use the new version here

Reply:
IMHO, let's go for the faster path and let's do the `sbt-org-policies` upgrade after this is out. Makes sense?